storage/store.go

```diff
@@ -472,7 +472,7 @@ func (s *Store) Start() error {
 		s.gossip.RegisterCallback(gossip.KeyConfigAccounting, s.configGossipUpdate)
 		s.gossip.RegisterCallback(gossip.KeyConfigZone, s.configGossipUpdate)
 		// Callback triggers on capacity gossip from all stores.
-		capacityRegex := fmt.Sprintf("%s.*", gossip.KeyMaxAvailCapacityPrefix)
+		capacityRegex := gossip.MakePrefixPattern(gossip.KeyMaxAvailCapacityPrefix)
```

Comment:
I agree that it doesn't look much clearer now, but I think it beats manually building regexes in a number of places that can only go up. I think without this we might just see weird bugs related to the fact that gossip keys can be prefixes of each other and then it matters that your regexes are solid (think node-1 vs node-count).